python/semantic_kernel/connectors/ai/open_ai/services/open_ai_chat_completion_base.py

```diff
@@ -301,9 +340,18 @@ async def _send_chat_stream_request(
     # endregion
     # region content creation
 
+    def _input_messages_to_prompt(self, messages: list[dict[str, Any]]) -> str:
```

Comment:
same for this, we have multiple ways of create a list of dict from messages, and settings have a list of dicts with the messages already here, i would argue that that is the list we want to log!